resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -839,16 +839,16 @@ private[spark] class Client(
     try {
       confStream.setLevel(0)
 
-      // Upload $SPARK_CONF_DIR/log4j.properties file to the distributed cache to make sure that
+      // Upload $SPARK_CONF_DIR/log4j2.properties file to the distributed cache to make sure that
       // the executors will use the latest configurations instead of the default values. This is
-      // required when user changes log4j.properties directly to set the log configurations. If
+      // required when user changes log4j2.properties directly to set the log configurations. If
       // configuration file is provided through --files then executors will be taking configurations
-      // from --files instead of $SPARK_CONF_DIR/log4j.properties.
+      // from --files instead of $SPARK_CONF_DIR/log4j2.properties.
 
       // Also upload metrics.properties to distributed cache if exists in classpath.
       // If user specify this file using --files then executors will use the one
       // from --files instead.
-      for { prop <- Seq("log4j.properties", "metrics.properties")
+      for { prop <- Seq("log4j2.properties", "metrics.properties")
```

Comment:
Do we need to consider the configuration file is `log4j2.xml` scenario?

Reply:
I will update with all possible configuration files.